app/views/partials/_navbar.html.haml

```diff
@@ -1,22 +1,24 @@
-%nav.flex.items-center.justify-between.flex-wrap.sunrise
-    .text-sm.lg:flex-grow.flex.justify-start.text-black.no-underline
-        =link_to image_tag('logo.png', {id: 'logo'}), root_path
+%nav
+    = link_to image_tag('logo.png', {id: 'logo'}), root_path
 
 -if user_signed_in?
-    = link_to 'Users', content_management_users_path if user_is_a_editor
-    = link_to 'Become a subscriber', new_subscriber_path(current_user) if user_is_a_member
-    = link_to 'New article', new_content_management_article_path unless user_is_a_member
-    = link_to 'Articles', content_management_articles_path if user_is_a_editor
-    = link_to 'Logout', destroy_user_session_path, method: :delete
+    .nav_button
+        .bg-white.text-sm.py-2.px-4.rounded
```

Comment:
Can you refactor this so that you're not styling every link?  Just use the .nav_button class in scss and style that.

Reply:
The refactoring kept breaking. I've set up a "Refactor styling" chore in PivotalTracker where I'll address any and all refactoring as a second step.